.vscode/settings.json

```diff
@@ -0,0 +1,44 @@
+{
+  // If the following is ever addressed, we can update our config to use whatever
+  // the official shared config is:
+  // https://github.com/microsoft/vscode/issues/40233
+
+  // This is required to load extensions, which in turn is required to be able
+  // to format SQL files.
+  // https://github.com/prettier/prettier-vscode/issues/3235
+  "prettier.configPath": "./.prettierrc.json",
```

Comment:
This was the key to getting SQL files formatting again; there's no need to install a separate extension.

Hopefully https://github.com/prettier/prettier-vscode/issues/3235 gets addressed soon.

Reply:
This only worked for me if I [downgrade the extension to 9.19.0](https://github.com/prettier/prettier-vscode/issues/3104#issuecomment-1681826376).